### ISSUE TITLE
Please let us set player executable path

### ISSUE THREAD

**Issue by barisahmet** [s1mpl3x/skadi#31]
I am using livestreamer "portable" way (in dropbox), and my video player path varies in different computers. It would be good if I can set player executable in the GUI (--player parameter in livestreamer).

**Comment by s1mpl3x** [comment]
older versions of skadi had a videoplayer config field, but i removed it as it seemed more clear to leave all that to livestreamer and i'd prefer to keep it that way. However adding a config value for the parameters to pass to livestreamer might be a possibility.

**Comment by barisahmet** [comment]
Yeah that is what I want, would be nice if I can pass --player to skadi :)

**Comment by s1mpl3x** [comment]
I've added a new config value to implement this:

``` xml
<skadi_data>
    [...]
    <livestreamer_args>
        <arg>--player=C:\Program Files (x86)\VideoLAN\VLC\vlc.exe</arg>
    </livestreamer_args>
</skadi_data>
```

this way it is possible to pass multiple args to livestreamer without doing any stupid string split fu. 
I don't want to add it to the gui atm, hope this works for you

**Comment by s1mpl3x** [comment]
[skadi-2.5.1-SNAPSHOT_7729993.zip](https://github.com/s1mpl3x/skadi/files/332410/skadi-2.5.1-SNAPSHOT_7729993.zip)

current dev snapshot, dont know when i'll release the next version

**Comment by barisahmet** [comment]
It should work, thanks man!
